core/src/storage_stage.rs

```diff
@@ -167,7 +167,7 @@ impl StorageStage {
                             ) {
                                 match e {
                                     Error::RecvTimeoutError(RecvTimeoutError::Disconnected) => {
-                                        break
+                                        break;
```

Comment:
wow, like really broken?

Reply:
yea, cargo fmt was adding that for me, I updated and it seems like it doesn't care now. will revert.